recipes/scorep/9.0_gcc12/gh200/extra/reframe.yaml

```diff
@@ -0,0 +1,11 @@
+default:
+  features:
+    - scorep
+    - scalasca
+    - mpi
+    - cuda
```

Comment:
Should this really provide these features? The package may accidentally be in the view, but they're not explicitly listed as packages?